tests/Integration/ResourceStatusSystem/Test_ResourceManagement.py

```diff
@@ -27,15 +30,15 @@ def test_addAndRemove():
   # ...............................................................................
 
   res = rsClient.addOrModifyAccountingCache('TestName12345', 'plotType', 'plotName', 'result', dateEffective, lastCheckTime)
-  assert res['OK'] == True
+  assert res['OK'] is True
```

Comment:
Yes. I had debugged that long ago. I don't remember where is this -O, but this is definitely the reason for that. This is also the reason why we use Exception and not assert in the WMSSecureGW.
see an example on a vobox with the dirac and the system pythons
```
[dirac@lbvobox108 log]$ which python
/opt/dirac/pro/Linux_x86_64_glibc-2.12/bin/python
[dirac@lbvobox108 log]$ python
Python 2.7.12 (default, Dec 17 2016, 22:41:37) 
[GCC 4.4.7 20120313 (Red Hat 4.4.7-17)] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> print __debug__
False
>>> assert False
>>> 

[root@lbvobox108 ~]# which python
/usr/bin/python
[root@lbvobox108 ~]# python
Python 2.6.6 (r266:84292, Aug 22 2016, 09:40:33) 
[GCC 4.4.7 20120313 (Red Hat 4.4.7-17)] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> print __debug__
True
>>> assert False
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AssertionError
>>>
```